src/js/components/Menu/stories/typescript/Children.tsx

```diff
@@ -30,12 +30,7 @@ const MenuWithChildren = (props) => (
 );
 
 const Example = () => (
-  <Box
-    align="center"
-    pad="large"
-    gap="small"
-    background={{ color: 'dark-3', opacity: 0.6 }}
-  >
+  <Box align="center" pad="large" gap="small" background="dark-1">
```

Comment:
Can we just remove the background on this box? We are trying to move away from using accent-, neutral-, light-, and dark- color references in an effort to make our storybook stories more compatible across different themes.

Reply:
Sure, I'll remove the background!